db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
+  {name: "My Community", abbrv: "MC"}
+])
+
+Household.create!([
+  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
+])
+
+User.create!([
+  {email: "<your gmail name>@gmail.com", google_email: "<your gmail name>@gmail.com", first_name: "Alice", last_name: "Admin", mobile_phone: "17345551212", household_id: 1, admin: true}
+])
+
+Resource.create!([
+  {name: "Dining Room", community_id: 1, default_calendar_view: "week", meal_abbrv: "CH", hidden: false}
+])
+
+
+
```

Comment:
I higly recommend you try out Sublime or Atom. Nobody uses Gedit :P

Also make sure whitespace chars are visible and you are auto-trimming
whitespace at end of lines.

On Fri, Sep 9, 2016 at 4:54 PM, Ken Winter <notifications@github.com> wrote:

> In db/seeds.rb.example
> <https://github.com/touchstone-cohousing/mess/pull/1#discussion_r78247332>
> :
>
> > +])
> > +
> > +Household.create!([
> > +  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
> > +])
> > +
> > +User.create!([
> > +  {email: "<your gmail name>@gmail.com", google_email: "<your gmail name>@gmail.com", first_name: "Alice", last_name: "Admin", mobile_phone: "17345551212", household_id: 1, admin: true}
> > +])
> > +
> > +Resource.create!([
> > +  {name: "Dining Room", community_id: 1, default_calendar_view: "week", meal_abbrv: "CH", hidden: false}
> > +])
> > +
> > +
> > +
>
> I'm about to check in this whole batch. If the new fake_user_data.rake
> file comes out wrong, let me know.
>
> FWIW, the editor I'm using is Gedit, the one that's bundled with Ubuntu.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/touchstone-cohousing/mess/pull/1/files/d49ab87ec9f05d1213b0a4b6bf614c19ebd67c5b#r78247332>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAq3cKBEa3HYIlKKHi-jQ4nfE7GOktpmks5qocdtgaJpZM4JzHfg>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech